brainzutils/musicbrainz_db/serialize.py

```diff
@@ -102,13 +135,16 @@ def serialize_recording(recording, includes=None):
 
 def serialize_places(place, includes=None):
     if includes is None:
-        includes = {}
+        includes = set()
```

Comment:
Hmm, this appears wrong. See Line 162 is trying to access this as a dict.

Reply:
yes, good catch. I thought that `includes` at this point was a list, and that `{}` was an incorrect attempt to make a set. however you're right, it should be a dict.

Reply:
the bug is in `serialize_series`, which sets it to a default value of a list... That's what would have confused me

Reply:
fixed, force pushed